### DISCUSSION TITLE
cut/copy/paste menu items does not work at all

### DISCUSSION THREAD

**Discussion by palansher** [thunderclient/thunder-client-support#637] [Q&A]
Hello!
I noticed that cut/copy/paste menu items does not work at all. I tried it on Environment names and data rows inside environment.
Is it my custom case only ? What is purpose of that copy/paste ? Nothing fount in README.

Thanks!

![image](https://user-images.githubusercontent.com/24850253/175827792-16487886-89eb-45a4-b420-b1f12cfbcba0.png)

![image](https://user-images.githubusercontent.com/24850253/175827803-ca60d47b-551a-4304-a6ab-98a3afac27a1.png)

**Comment by rangav** [comment]
The cut/copy menu is provided by vscode not thunder Client.

it will work if you can select any text, not on thunder Client menu.

**Reply by palansher** [reply]
Got it, Thanks!

It was really confusing :)
Great you made mouse drag-and-drop possibility to move items through collections.
